source/getting-started/admin-onboarding-tasks.rst

```diff
@@ -87,7 +87,7 @@ You can enable plugins and integrations to connect your team's workflows and too
 To enable and manage plugins, go to **System Console > Plugins**. Next, install plugins with a single click from the **Main Menu > Plugin Marketplace**. See the `Plugin Marketplace  <https://docs.mattermost.com/administration/plugins.html#plugin-marketplace>`__ documentation for details.
 
 Consider installing and enabling the following productivity plugins for your users:   
-  - Enable audio/video calling and screen sharing with `Jitsi <https://integrations.mattermost.com/?s=Jitsi%20Plugin>`__ or `Zoom <https://integrations.mattermost.com/?s=Zoom%20Plugin>`__.
+  - Enable audio/video calling and screen sharing with `Jitsi <https://mattermost.com/blog/mattermost-and-jitsi/>`__ or `Zoom <https://mattermost.com/marketplace/zoom-plugin/>`__.
   - Create polls with `Matterpoll <https://integrations.mattermost.com/?s=Matterpoll%20Plugin>`__.
```

Comment:
Curious as to why the following three links weren't changed along with the previous two?

Reply:
@krutarththakkar, would you be open to updating all the links here so they have the correct URLs (i.e., do not include the search string but are the direct URL). Thank you. :)

Reply:
Ok, Am on it!